css/styles.css

```diff
@@ -82,3 +87,70 @@ body {
   color: #fff;
   background: #2230d2;
 }
+
+/* css for desktop version */
+
+@media only screen and (min-width: 768px) {
+  .container {
+    height: 916px;
+    background: #fff;
+  }
```

Comment:
There seems to be an overflow on the page. Here's the website on a 1285px screen Kindly fix this and _ensure that the mobile version should be responsive until the desktop screen size._

![image](https://user-images.githubusercontent.com/20554583/206481181-2642242a-b23d-4377-9a14-3653af4bf5b4.png)